frontend/src/routes/Bookshelf.jsx

```diff
@@ -18,16 +24,48 @@ function Bookshelf({ backend, user }) {
     .map(({_id, dc_title, links}) => [_id, dc_title, links]);
   const displayedDocs = docs?.flatMap(d => d[0]);
 
+  function DisplayDocuments() {
+    if (displayMode === 'graph') {
```

Comment:
Because conditions are on the same variable, you may use a `switch` structure instead...

Reply:
Thanks!